.github/workflows/sentry-pull-request-bot.yml

```diff
@@ -0,0 +1,61 @@
+name: sentry pull request bot
+
+# Note this event happens on Issue comments AND PR comments,
+# we make sure that we only respond to PR comments.
+on:
+  issue_comment:
+    types: [created, edited]
+  pull_request:
+    types: [opened, edited]
+
+jobs:
+  # TODO(billy): Move this into an external action as we add more functionality
+  test-getsentry:
+    name: test getsentry
+    runs-on: ubuntu-16.04
+
+    # Ensure this bot only responds for pull requests and only for the main repository
+    if: >-
+      (github.event.issue.pull_request.url != '' || github.event.pull_request.id != '') &&
+      (contains(github.event.comment.body, '#test-getsentry') || contains(github.event.pull_request.body, '#test-getsentry')) &&
+      github.repository == 'getsentry/sentry'
+
+    steps:
+      - name: Check getsentry membership
+        id: org
+        uses: actions/github-script@v3
+        with:
+          script: |
+            try {
+              const result = await github.orgs.checkMembershipForUser({
+                org: 'getsentry',
+                username: context.payload.sender.login,
+              })
+              return result.status == 204;
+            } catch {
+              return false;
+            }
+
+      - name: Fetch getsentry token
+        if: steps.org.outputs.result == 'true'
+        id: getsentry
+        uses: getsentry/action-github-app-token@v1
+        with:
+          app_id: ${{ secrets.SENTRY_INTERNAL_APP_ID }}
+          private_key: ${{ secrets.SENTRY_INTERNAL_APP_PRIVATE_KEY }}
+
+      - name: Dispatch getsentry tests
+        if: steps.org.outputs.result == 'true'
+        uses: actions/github-script@v3
+        with:
+          github-token: ${{ steps.getsentry.outputs.token }}
+          script: |
+            github.actions.createWorkflowDispatch({
+              owner: 'getsentry',
+              repo: 'getsentry',
+              workflow_id: 'acceptance.yml',
```

Comment:
In the future, are you planning on multiple workflow_ids? Or maybe "all" by default. This is definitely a better experience than the manual 1) open getsentry PR, 2) manually bump the correct sentry commit, but perhaps the first step towards a comment-based workflow. Wonder how bors setup compares in difficulty and extensibility.

Reply:
At the moment there are no plans for multiple workflow_ids - I'm thinking we can add more workflows as needed, but currently, javascript changes are generally the biggest cause of build failures.

Yeah I would like to peek into the extensibility of bors, as that is probably a solution we will want.